dispatcher/dispatcher.go

```diff
@@ -1252,3 +1255,66 @@ func (d *Dispatcher) setNodeRole(id NodeId, role OtDeviceRole) {
 	node.Role = role
 	d.vis.SetNodeRole(id, role)
 }
+
+func (d *Dispatcher) handleCoapEvent(node *Node, argsStr string) {
+	var err error
+
+	if d.coaps == nil {
+		// Coaps not enabled
+		return
+	}
+
+	args := strings.Split(argsStr, ",")
+
+	simplelogger.AssertTrue(len(args) > 0)
```

Comment:
I think we should not crash on malformed incoming events. What about logging with `Error`/`Alert` level?

Reply:
Generally I agree that otns should not crash on malformed inputs. However, here OTNS is processing inputs from OpenThread, which should never generate malformed output.
I would like OTNS to crash loudly so that we can easily catch and fix bugs.
Thoughts?